Jenkinsfile

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/groovy
-@Library('github.com/fabric8io/fabric8-pipeline-library@master')
+@Library('github.com/kishansagathiya/fabric8-pipeline-library@remove-content-repo')
```

Comment:
@kishansagathiya I guess we need to change it back to github.com/fabric8io/fabric8-pipeline-library@master.
Right?